internal/binutils/binutils.go

```diff
@@ -163,6 +163,14 @@ func (bu *Binutils) Disasm(file string, start, end uint64) ([]plugin.Inst, error
 		fmt.Sprintf("--start-address=%#x", start),
 		fmt.Sprintf("--stop-address=%#x", end),
 		file)
+
+	if runtime.GOOS == "darwin" {
```

Comment:
So without this disasm doesn't work on Mac at all? If that's the case perhaps we should put this on its own PR

Reply:
https://github.com/google/pprof/pull/534